app/models/solidus_subscriptions/subscription.rb

```diff
@@ -380,13 +376,6 @@ def emit_events_for_update
           subscription: self,
         )
       end
-
-      if previous_changes.key?('payment_source_id') || previous_changes.key?('payment_source_type') || previous_changes.key?('payment_method_id')
-        ::Spree::Event.fire(
-          'solidus_subscriptions.subscription_payment_method_changed',
-          subscription: self,
-        )
-      end
```

Comment:
We should keep all the `Spree::Event.fire` calls in place. Users of the extension can listen to those events to fire their own logic — they are not specific to the ChurnBuster integration, even though the integration did make use of them.

Reply:
Done!